subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CachedCustomTaskExecutionIntegrationTest.groovy

```diff
@@ -626,7 +626,7 @@ class CachedCustomTaskExecutionIntegrationTest extends AbstractIntegrationSpec i
         when:
         withBuildCache().run "customTask", "--info"
         then:
-        output.contains "Caching disabled for task ':customTask': Invalid build cache key was generated"
```

Comment:
We should print the task class here, too, not just with actions. Actually, since actions are most probably gonna be closures from the build script, printing task types will be more useful than those.

Reply:
The action will not be closures from a build script, since closures from a build script will have the correct classloader and won't be lambdas.

I think that it is not necessary to print the class name of a task, since that is easily discoverable from the command line - run `gradle help --task <taskName>`.

Reply:
OK. I'd still print the task name as `gradle help --task` is anything but easily discoverable.

Reply:
We are printing the task class now.

Reply:
Should we check that in the test?

Reply:
Checking now.